codespell_lib/data/dictionary.txt

```diff
@@ -10816,14 +10931,23 @@ coctail->cocktail
 cocument->document
 cocumentation->documentation
 cocuments->documents
+cocurrency->concurrency
+cocurrent->concurrent
+cocurrently->concurrently
 codde->code, coded, coddle,
+codder->coder
```

Comment:
Perhaps `coddler`? Probably rare, but that would be consistent with `codded->coded, coddled,`.